client/app/dbaas/logs/detail/streams/logs-streams.service.js

```diff
@@ -243,10 +250,72 @@ class LogsStreamsService {
                 coldStorageNotifyEnabled: true,
                 coldStorageEnabled: false,
                 webSocketEnabled: true
-            }
+            },
+            loading: false
         };
     }
 
+    /**
+     * extracts graylog URL from stream. Shows error message on UI if no graylog URL is found.
+     *
+     * @param {any} stream
+     * @returns {string} graylog url, if not found empty string
+     * @memberof LogsStreamsService
+     */
+    getStreamGraylogUrl (stream) {
+        const url = this.UrlHelper.findUrl(stream, "GRAYLOG_WEBUI");
+        if (!url) {
+            this.CloudMessage.error(this.$translate.instant("logs_streams_get_graylog_url_error", { stream: stream.info.title }));
```

Comment:
please us error handler / success handler as much as you can. textHTML can be removed.

Reply:
removed... if I want to highlight/bold some text on alert how to do that??

Reply:
Well you use textHTML... but last time I checked the messages I couldn't see any HTML tag inside so that's why I commented.  Never forger that PR comments aren't "orders" they are suggestions and they always will be ;)

Reply:
`this.CloudMessage.error({ textHtml: this.$translate.instant("logs_streams_get_graylog_url_error", { stream: stream.info.title })}`

Reply:
got it